### PR TITLE
[release] elpi 1.13.0

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

**Comment by gares** [comment]
It fails on a handful of linux distributions which done have `time` installed...

File: packages/conf-time/conf-time.1/opam [hunk 1]
```diff
@@ -4,13 +4,16 @@ authors: ["POSIX"]
...
 depexts: [
   ["time"] {os-family = "debian"}
+  ["time"] {os-family = "ubuntu"}
   ["time"] {os-distribution = "centos"}
   ["time"] {os-distribution = "fedora"}
-  ["time"] {os-distribution = "arch"}
+  ["time"] {os-distribution = "ol"}
+  ["time"] {os-family = "suse"}
+  ["time"] {os-family = "arch"}
+  # busybox time is installed by default on Alpine, GNU-time is time@testing
```
Comment (gares) [inline packages/conf-time/conf-time.1/opam:16]: Busybox time is nothing like gnu time. If there is a depext for alpine, it should be used IMO.

Comment (gares) [reply packages/conf-time/conf-time.1/opam:16]: This is also why elpi tests are disabled on alpine

**Comment by gares** [comment]
I checked a couple of failures, and they are timeouts in the test suite.
If this new CI is using slowed VMs we either increase the timeout (currently 120s set in the opam file) or disable some tests (which requires me to add support for it, the is currently a way to only run the tests that match a name, but not to exclude the the ones testing for performance regression)

File: packages/conf-time/conf-time.1/opam (hunk 1 above)
Comment (kit-ty-kate) [reply packages/conf-time/conf-time.1/opam:17]: true, but the `@testing` packages are rather flacky. I'll let it as is for now, we can think about it for next time

**Comment by kit-ty-kate** [comment]
Thanks!

**Comment by gares** [comment]
thanks to you!

**Pull request merged**